vector/src/main/res/values/strings.xml

```diff
@@ -352,7 +352,7 @@
     <string name="sign_out_bottom_sheet_dont_want_secure_messages">I don’t want my encrypted messages</string>
     <string name="sign_out_bottom_sheet_backing_up_keys">Backing up keys…</string>
     <string name="keys_backup_activate">Use Key Backup</string>
```

Comment:
It is done for the ponctuation. 
For translation I think it should be handled in an issue which group all translation verification.

Reply:
Okay I don't know really the translation process when it is just ponctuation changes. @bmarty what is the process for such changes?

Reply:
According to the [doc](https://github.com/vector-im/element-android/blob/main/CONTRIBUTING.md#editing-existing-strings), it's OK to edit the string. We are in case 1:

> If the meaning stays the same, it's OK to edit the original string (i.e. the English version).

What is not described in the doc, is that Weblate will detect automatically the change and mark all the translations as "need update" (or something similar). So we will see updates of those translations in the future PRs from Weblate.